### PR TITLE
fix: bug for result - helm get <name>

### PR DESCRIPTION
**Pull request by RobinCK**

### CODE REVIEW

File: src/services/K8sHelmService.ts [hunk 1]
```diff
@@ -255,10 +249,19 @@ export class K8sHelmService {
             computedValues = yaml.safeLoad(values.join('\n')) || {};
         }
 
+        const searchValue = (text: string, label: string) => {
+            const value = text.split(new RegExp(`^${label}:\s*(.+?)^[A-Z]{2,}`, 'gmus'))[1];
```
Comment (vlad-tkachenko) [inline src/services/K8sHelmService.ts]: what is `s` flag?

Comment (vlad-tkachenko) [reply src/services/K8sHelmService.ts]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp

File: src/services/K8sHelmService.ts [hunk 2]
```diff
@@ -255,10 +249,19 @@ export class K8sHelmService {
...
+
+            if (value) {
+                return value.replace(/\n/gs, '');
```
Comment (vlad-tkachenko) [inline src/services/K8sHelmService.ts]: Again, `s` flag here.

File: src/services/K8sHelmService.ts (hunk 1 above)
Comment (RobinCK) [reply src/services/K8sHelmService.ts]: flag `s`: Single line - Dot matches newline

File: src/services/K8sHelmService.ts [hunk 3]
```diff
@@ -202,63 +203,18 @@ export class K8sHelmService {
...
-                }
-            }
-
-            userSuppliedValues = yaml.safeLoad(values.join('\n')) || {};
-        }
-
-        if (computedValuesLine) {
-            let idx = lines.indexOf(computedValuesLine);
-            const values = [];
+        const matches = helmResult.stdout.split(/^(\b[A-Z]{2,}\s?.*?):/gm);
```
Comment (vlad-tkachenko) [inline src/services/K8sHelmService.ts]: `/^([A-Z]{2,}[^:]+):/gm`

**Pull request merged**